code/modules/lore_codex/legal_code_data/main.dm

```diff
@@ -0,0 +1,99 @@
+/datum/lore/codex/category/main_corp_regs // The top-level categories for SOP/Regs/Law/etc
```

Comment:
Again, this needs to go away. Sif Governmental Authority is not a thing in our lore and nor is the SCG.